Source/ACE.Server/WorldObjects/Healer.cs

```diff
@@ -263,18 +268,11 @@ public uint GetHealAmount(Player healer, Player target, CreatureVital vital, out
             var healMax = healBase * 0.5f;
             var healAmount = ThreadSafeRandom.Next(healMin, healMax);
 
-            // verify healing boost comes from target instead of healer?
-            // sounds like target in LumAugHealingRating...
-            var ratingMod = target.GetHealingRatingMod();
-
-            healAmount *= ratingMod;
-
             // chance for critical healing
             criticalHeal = ThreadSafeRandom.Next(0.0f, 1.0f) < 0.1f;
             if (criticalHeal) healAmount *= 2;
 
             // cap to missing vital
-            var missingVital = vital.MaxValue - vital.Current;
```

Comment:
should be ``var missingVital = Math.Min(vital.Missing, prevMissingVital);`` i think

if the player gets healed from another source while they are performing the healing animation, it seems like the pre-cap to max vital could have an overage without this line

Reply:
That is intentional. The "Receiving healing (spell) during animation:" timestamps in the commit comment show that the displayed heal amount was not reduced by any healing received during the animation. It also appears that the stamina consumption is in line with the displayed amount and not the actual amount. 
Personally I think it's a dumb retail bug and would have no issues correcting the displayed amount to the actual amount healed, but I suppose it needs to stay that way to be retail accurate.